environments/staging/app-processes.yml

```diff
@@ -34,43 +34,51 @@ pillows:
       num_processes: 1
     CacheInvalidatePillow:
       num_processes: 1
-    CaseSearchToElasticsearchPillow:
-      num_processes: 1
-    CaseToElasticsearchPillow:
-      num_processes: 1
+    # CaseSearchToElasticsearchPillow:
+    #   num_processes: 1
+    # CaseToElasticsearchPillow:
+    #   num_processes: 1
     DefaultChangeFeedPillow:
       num_processes: 1
     DomainDbKafkaPillow:
       num_processes: 1
-    FormSubmissionMetadataTrackerPillow:
-      num_processes: 1
-    GroupPillow:
-      num_processes: 1
-    GroupToUserPillow:
-      num_processes: 1
-    kafka-ucr-main:
-      num_processes: 1
-    kafka-ucr-static:
-      num_processes: 1
+    # FormSubmissionMetadataTrackerPillow:
+    #   num_processes: 1
+    # GroupPillow:
+    #   num_processes: 1
+    # GroupToUserPillow:
+    #   num_processes: 1
+    # kafka-ucr-main:
+    #   num_processes: 1
+    # kafka-ucr-static:
+    #   num_processes: 1
     KafkaDomainPillow:
       num_processes: 1
     LedgerToElasticsearchPillow:
       num_processes: 1
-    ReportCaseToElasticsearchPillow:
-      num_processes: 1
-    ReportXFormToElasticsearchPillow:
-      num_processes: 1
+    # ReportCaseToElasticsearchPillow:
+    #   num_processes: 1
+    # ReportXFormToElasticsearchPillow:
+    #   num_processes: 1
     SqlSMSPillow:
       num_processes: 1
-    UnknownUsersPillow:
-      num_processes: 1
+    # UnknownUsersPillow:
+    #   num_processes: 1
     UpdateUserSyncHistoryPillow:
       num_processes: 1
     UserCacheInvalidatePillow:
       num_processes: 1
     UserGroupsDbKafkaPillow:
       num_processes: 1
-    UserPillow:
-      num_processes: 1
-    XFormToElasticsearchPillow:
+    # UserPillow:
+    #   num_processes: 1
+    # XFormToElasticsearchPillow:
```

Comment:
These should be removed instead of commented out

Reply:
Sure, I was keeping it in case I had to roll back :-|. Will remove it now :)